homeassistant/components/climate/__init__.py

```diff
@@ -25,6 +25,7 @@
 DEFAULT_MAX_TEMP = 35
 DEFAULT_MIN_HUMITIDY = 30
 DEFAULT_MAX_HUMIDITY = 99
+DEFAULT_TEMP_STEP = 1
```

Comment:
Don't add default to the climate component, keep it in MQTT component.

Reply:
Removed. Should I squash the commits or is it fine this way?